tests/test_fieldset_sampling.py

```diff
@@ -184,16 +184,28 @@ def test_nearest_neighbour_interpolation3D(mode, k_sample_p, npart=81):
 
 
 @pytest.mark.parametrize('mode', ['scipy', 'jit'])
-def test_fieldset_sample_particle(fieldset, mode, k_sample_uv, npart=120):
+@pytest.mark.parametrize('lat_flip', [False, True])
```

Comment:
should we also check for curvilinear grids, explicitly?

Reply:
Actually, I think we shouldn't flip at all curvi grids.
Nothing requires to have ascending latitudes in that case.

So I would remove the flip from that class.